src/Symfony/Component/HttpFoundation/RequestMatcher.php

```diff
@@ -51,18 +57,20 @@ class RequestMatcher implements RequestMatcherInterface
     /**
      * @param string|null          $path
      * @param string|null          $host
+     * @param int|null             $port
      * @param string|string[]|null $methods
      * @param string|string[]|null $ips
      * @param array                $attributes
      * @param string|string[]|null $schemes
      */
-    public function __construct(string $path = null, string $host = null, $methods = null, $ips = null, array $attributes = array(), $schemes = null)
+    public function __construct(string $path = null, string $host = null, $methods = null, $ips = null, array $attributes = array(), $schemes = null, $port = null)
```

Comment:
should be `int $port = null`

Reply:
Oh yeah, now we can use typehint in this version :+1: